CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   - K8sRole | Annotation | Role Annotation
   - K8sRoleBinding | Label | Role Binding Label
   - K8sRoleAnnotation | Annotation | Role Binding Annotation
+- ROX-12496: CSV export API endpoint `/api/vm/export/csv` will is deprecated and will be removed in 3.73. It will be replaced with three new endpoints, one for each CVE type.
```

Comment:
`will is deprecated` -> `is deprecated`
`3.73` -> ` in the future` because it depends on postgres GA.

We continue to support features that are in the deprecation phase.